paper/content/chapter2/chapter2_1.tex

```diff
@@ -1,9 +1,156 @@
 % !TeX root = ../../paper.tex
-\subsection{Evaluierungsfunktion}
+\subsection{Bewertungsfunktion}
 
+In der Spieltheorie ist es in der Regel nicht möglich, alle möglichen Zugfolgen aus einer Spielposition heraus bis zum Ende zu verfolgen.
+Deshalb \glqq [...] wird eine Funktion benötigt, die die Stellung auf dem Spielbrett danach bewertet,
+ob sie für eine der beiden Parteien vorteilhaft oder nachteilig ist.\grqq \ [\cite{Paulsen2009}]
+Diese Funktion wird als \textit{Bewertungsfunktion} bezeichnet.
+Die Bewertungsfunktion setzt sich aus einem materiellen und einer positionellen Komponente zusammen.
+Bei der materiellen Komponente werden die verbleibenden Figuren auf der eigenen Seite gezählt und die Anzahl der gegnerischen Figuren von diesem Wert subtrahiert.
+Daraus lassen sich erste Schlussfolgerungen über den Spielstand ziehen.
+Zudem ist es möglich, die derzeitige Spielphase abzuleiten [\cite{Paulsen2009}].
 
-\subsubsection{Einfache Evaluierungsfunktion}
+Da es beim Schach aber auch entscheidend ist, auf welchen Positionen sich die einzelnen Figuren befinden und in welcher Position sie zueinander stehen (Bauernstruktur, Königssicherheit), wird zusätzlich zur materiellen Komponente eine positionelle Komponente berechnet.
+Die verschiedenen Positionen werden aus der Spielbrettaufstellung entnommen und fließend mit ihren jeweiligen Gewichtungen in die anschließende Bewertung ein [\cite{Paulsen2009}].
 
 
-\subsubsection{Weitere Evaluierungsfunktionen}
+\subsubsection{Einfache Bewertungsfunktion}
+
+Das Schachspiel setzt sich aus zwei wichtigen Faktoren zusammen: Einerseits ist abgesehen von der Tatsache, welcher Spieler den ersten Zug macht, keine Zufallskomponete enthalten, andererseits handelt es sich um ein Spiel mit \gls{PI}.
+Diese zwei Faktoren führen dazu, dass bei jeder Schachbrettposition eine der folgenden drei Aussagen gilt [\cite{Shannon1950}]:
+
+\begin{enumerate}
+    \item Es handelt sich um eine gewonnene Position für Weiß. Somit kann Weiß einen Sieg forcieren, wobei Schwarz verteidigen.
+    \item Es handelt sich um eine gewonnene Position für Schwarz. Somit kann Schwarz einen Sieg forcieren, wobei Weiß spielt.
+    \item Es handelt sich um eine unentschiedene Position für beide Parteien. Somit kann es nur ein Unentschieden am Ende geben, falls beide Parteien dies forcieren und keine Fehler machen.
+\end{enumerate}
+
```

Comment:
Oftmals gibt es auch noch zwei weitere Möglichkeiten: 
1. Vorteil für weiß: Weiß hat einen deutlichen Vorteil, kann jedoch in absehbarer Zukunft keinen Sieg forcieren
2. Vorteil für schwatz: Schwarz " "

Mittlere Möglichkeit würde ich nicht mit "unentschieden" bezeichnen, sondern mit "ausgeglichen"

Reply:
Ersteres bräcuhten wir noch eine Quelle zu. Den Rest passe ich an.

Reply:
Okay habe den Quelltext gelesen in dem Zusammenhang macht es mit 3 Möglichkeiten mehr Sinn, allerdings würde  ich erwähnen, dass es mit der heutigen computing power nicht Möglich ist, immer den besten Zug zu bestimmen, also dass dieses Modell praktisch keinen Nutzen hat. (Wäre auch eine gute Überleitung zu dem Punkt, dass wir eine andere eval function als die basic -1 , 0 , +1 verwenden)

Reply:
hat der Shannon ja auch geschrieben (das mit den 10^120 und 10^43)